app/assets/javascripts/webchat/library.js

```diff
@@ -91,9 +91,13 @@
 
     function advisorStateChange (state) {
       state = state.toLowerCase()
-      var currentState = $el.find('.' + webchatStateClass + state)
-      $el.find('[class^="' + webchatStateClass + '"]').addClass('govuk-!-display-none')
-      currentState.removeClass('govuk-!-display-none')
+      var currentState = el.querySelectorAll('[class^="' + webchatStateClass + state + '"]')[0]
```

Comment:
In jQuery, `.find()` finds everything that matches. In JavaScript, it's weirdly more complex

- `querySelector` finds the first element that matches
- `querySelectorAll` finds all elements that match

If you do `querySelectorAll()[0]` that returns the first one found - it's equivalent to doing `querySelector`. Which one do you want to do here? I suspect it's `querySelector`?

Reply:
🤦‍♀️  Yes, I just need `querySelector`